src/Services/SearchService.php

```diff
@@ -275,7 +275,10 @@ private function shouldRedirectToProductDetailPage(array $productsIds): bool
         return $type !== 'corrected' && $type !== 'improved';
     }
 
-    private function getProductDetailUrl(int $productId): ?string
+    /**
+     * @return string|null
```

Comment:
Why did adding `?string` not work?

Reply:
Using : ?string throws an error when provisioning plugin on Plentymarkets system.
233: UndefError SyntaxError syntax error, unexpected '?'

Same problem as in the https://github.com/findologic/plugin-plentymarkets-ceres-api/pull/142

Reply:
Ah yes I have totally forgotten that. Damn.